packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderCellExample.shorthand.tsx

```diff
@@ -0,0 +1,19 @@
+import { Datepicker, Tooltip } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerHeaderCellExample = () => {
+  return (
+    <Datepicker
+      today={new Date(2020, 8, 12, 0, 0, 0, 0)}
+      calendar={{
+        calendarHeaderCell: {
+          children: (ComponentType, props) => (
+            <Tooltip content={props['aria-label']} trigger={<ComponentType {...props} />} />
```

Comment:
It looks that we will have `aria-label` on a `trigger` still, is it expected?

Reply:
Yep :)